src/main/java/org/osc/controller/nsc/utils/InspectionHookElementImpl.java

```diff
@@ -0,0 +1,92 @@
+package org.osc.controller.nsc.utils;
+
+import org.osc.sdk.controller.FailurePolicyType;
+import org.osc.sdk.controller.TagEncapsulationType;
+import org.osc.sdk.controller.element.InspectionHookElement;
+import org.osc.sdk.controller.element.InspectionPortElement;
+import org.osc.sdk.controller.element.NetworkElement;
+
+public class InspectionHookElementImpl implements InspectionHookElement {
```

Comment:
Can we make the entities implement these *Element interfaces this way we dont need the *ElementImpl types?

Reply:
Done